README.md

```diff
@@ -361,6 +361,29 @@ To build standalone source code package, the following commands can be issued:
 
 After that, a stanalone archive with source code will be created in the `.build` directory.
 
+When cross compiling, the AS, AR, CC, CXX, LD, etc. variables should be set according to the target/cross compile toolchain. Additional variables should be configured:
+  * set the ARCHITECTURE option to the target architecture
+  * set the CROSS_COMPILE option to 1
+  * set HOST_* options for the build host (HOST_AS, HOST_AR, HOST_CC, HOST_CXX, HOST_LD, HOST_CFLAGS, HOST_CXXFLAGS, HOST_EXE_FLAGS, HOST_SO_FLAGS)
+  * set the PREFIX option to a non-system directory
+
+Example cross compile procedure for aarch64 target on x86-64 build host:
+```
+  make clean
+  make config ARCHITECTURE="aarch64" CROSS_COMPILE="1" \
+    HOST_AS="as" HOST_AR="ar" HOST_CC="gcc" HOST_CXX="g++" HOST_LD="ld" \
```

Comment:
What are the default values of these variables when you cross-compile the source code?
According to the `tools.mk` file, they are set the same to `$(AS)`, `$(AR)`, `$(CC)`, `$(CXX)`, etc.

Reply:
The problem is that normally a cross compile toolchain will override AS, AR, CC, CXX, etc with the cross compiler tools (AR=aarch64-linux-ar, CC=aarch64-linux-gcc, CXX=aarch-linux-g++, ...). And then the HOST_* versions of those variables are being set to the same as the cross compiler versions.